### DISCUSSION TITLE
Get all ids of entity indexby another column

### DISCUSSION THREAD

**Discussion by SebastienLcmt** [doctrine/doctrine2#11055] [Support Questions]
Hello everyone, I would please appreciate some help on what seemed like a small problem.
I have an entity s with a 'name' column, and all I am trying to do is to get all ids indexed By that name column (['name1' => id1, 'name2 => id2, etc]). So a good old associative array. 
I have tried A LOT of things but i can't seem to get it to work. I won't get the name column unless i select it, but then I get something like
['name1' => ['id' => 1, 'name' => name1].
Which is not what I want.
Thank you in advance for your time. 
```php
$results = $this->createQueryBuilder('s', 's.name')
        ->select('s.id')
        ->getQuery()
        ->getResult()
;
```

**Comment by lrdvdr** [comment]
any updates on this?
